core/templates/dev/head/domain/exploration/StateObjectFactory.js

```diff
@@ -46,6 +46,20 @@ oppia.factory('StateObjectFactory', [
       };
     };
 
+    State.createDefaultState = function(newStateName) {
```

Comment:
We probably want to move this file out of /exploration, since it's going to be shared. Perhaps to /state?

Reply:
Done.